test/functional/Java9andUp/build.xml

```diff
@@ -22,7 +22,7 @@
   SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-<project name="Test Example" default="build" basedir=".">
+<project name="Java9andUp" default="build" basedir=".">
 	<taskdef resource='net/sf/antcontrib/antlib.xml'/>
 	<description>
 		Test Example
```

Comment:
Do you also want to update the description text?

Reply:
Thanks for pointing out. The changeset is updated.